packages/apps-config/src/api/spec/interbtc.ts

```diff
@@ -49,10 +49,11 @@ export function getBalance (
   return memo(
     instanceId,
     (account: string): Observable<DeriveBalancesAll> =>
-      combineLatest<[any]>([api.query.tokens.accounts(account, { Token: nativeToken })]).pipe(
-        map(([data]: [OrmlAccountData]): DeriveBalancesAll => {
+      combineLatest<[any, any]>([api.query.tokens.accounts(account, { Token: nativeToken }), api.query.system.account(account)]).pipe(
+        map(([data, systemAccount]: [OrmlAccountData, any]): DeriveBalancesAll => {
           return {
             ...defaultAccountBalance(),
+            accountNonce: new BN(systemAccount.nonce),
```

Comment:
```suggestion
            accountNonce: api.registry.createType('Index', systemAccount.nonce),
```

I guess the (build) issue is that we have `any` here, hence it doesn't know what to do.

The more-correct way is probably to -

```js

import { FrameSystemAccountInfo } from '@polkadot/types/lookup';

...
map(([data, systemAccount]: [OrmlAccountData, FrameSystemAccountInfo]): DeriveBalancesAll => {
```

So the typings are just correct and no faffing around with createType

Reply:
Thanks for the pointer! I updated the PR